src/proposal.c

```diff
@@ -3444,6 +3560,25 @@ int Move_ExtSPRClock (Param *param, int chain, RandLong *seed, MrBFlt *lnPriorRa
             brlens = GetParamSubVals (subParm, chain, state[chain]);
             brlens[a->index] = (rate[a->index] + rate[b->index]) / 2.0 * (a->length + u->length);
             }   /* end tk02 branch rate parameter */
+        else if ( subParm->paramType == P_WNBRANCHRATES)
+            {
+            if (subParm->paramType == P_WNBRANCHRATES)
+                var = *GetParamVals (modelSettings[subParm->relParts[0]].wnvar, chain, state[chain]);
+            else
+                var = *GetParamVals (modelSettings[subParm->relParts[0]].mixedvar, chain, state[chain]);
```

Comment:
Looks like copy and paste without correction, see above. All four lines should probably be replaced by line 3566.